serverless/serverless-ha.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-ha"]
+= High availability on {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: serverless-ha
+
+toc::[]
+
+High availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
+In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving or Eventing control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
+These controller instances compete to use a shared resource, known as the leader election lock.
+The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+== Configuring high availability replicas on {ServerlessProductName}
+
+High availability (HA) functionality is available by default on {ServerlessProductName} for the `autoscaler-hpa`, `controller`, `activator`, `kourier-control`, and `kourier-gateway` controllers. These components are configured with two replicas by default.
+
+You modify the number of replicas that are created per controller by changing the configuration of `KnativeServing.spec.highAvailability` in the KnativeServing custom resource definition (CRD) and `KnativeEventing.spec.highAvailability` spec in the KnativeEventing CRD.
```

Comment:
This bit looks like a copy of the bits in serverless/admin_guide/serverless-ha.adoc , so the same comment as above applies here...

Reply:
@maschmid My bad, removed the duplicate files.